Website/plugins/secondaries/gen-secondaries.raku

```diff
@@ -123,7 +125,7 @@ sub ($pp, %processed, %options) {
                 :meta(''),
                 :footnotes(''),
             ), %templates);
-            @transfers.push: ["$fn-name\.html", 'myself', "html/$fn-name\.html"]
+            @transfers.push: ["$fn-name\.html", 'myself', "html/$fn-name.html"]
```

Comment:
The backslash was removed in front of the dot but only at one place. Which one is correct?

Reply:
done